src/calendar/index.ts

```diff
@@ -324,20 +343,29 @@ export class CalendarBase<P extends CalendarProperties = CalendarProperties> ext
 		return weeks;
 	}
 
-	protected renderDateCell(date: number, index: number, selected: boolean, currentMonth: boolean, today: boolean): DNode {
+	protected renderDateCell(dateObj: Date, index: number, selected: boolean, currentMonth: boolean, today: boolean): DNode {
+		const {
+			minDate = dateObj,
+			maxDate = dateObj
+		} = this.properties;
+
+		const date = dateObj.getDate();
 		const { theme, classes } = this.properties;
+		const outOfRange = (dateObj < minDate || dateObj > maxDate);
```

Comment:
how about conditionally running this check rather than defaulting the min / max values above?

Reply:
Sure, I can change this. I wen't back and forth on how to write it. This was a little more legible but might be slightly less performant.